R/bounds.r

```diff
@@ -155,17 +155,52 @@ rescale_none <- function(x, ...) {
   x
 }
 
-#' Censor any values outside of range
+#' @name oob
+#' @aliases oob_censor censor oob_squish_finite oob_squish_any squish
+#'   oob_squish_infinite squish_infinite oob_discard discard oob_keep
+#' @title Out of bounds handling
+#' @description This set of functions modify data values outside a given range.
+#'
+#' @param x A numeric vector of values to modify.
+#' @param range A numeric vector of length two giving the minimum and maximum of
+#'   the desired output range respectively.
+#' @param only.finite A logical of length one, which when `TRUE` will only alter
```

Comment:
I realize this is just copied over from the previous version, but I don't like the phrasing here. As written, the sentence implies that the parameter `only.finite` alters the data, which is not true. It's the function that alters the data. A simple way to fix this may be to use passive voice: "A logical of length one. When set to `TRUE` only ... will be altered ..."